### ISSUE TITLE
requestPermission method not returning value, kind of struct this line

### ISSUE THREAD

**Issue by Anand8293** [matinzd/react-native-health-connect#176]
**Describe the bug**

**To Reproduce**
1. Go to call "requestPermission"

```ts
try {
      const result = await requestPermission([
        {
          accessType: 'read',
          recordType: 'Steps',
        },
      ]);
      **console.log('Permission Result:', result);** Not printing value
    } catch (error) {
      console.error('Permission Error:', error);
    }
```
    

**Expected behavior**  
Should return permission lists

**Environment:**
 - Health Connect Version: 3.3.0
 - React Native Version: 0.72.4
 - New architecture enabled: Yes
 - Using Expo: No/Go/Prebuild(Dev Client)
 - Android API Level: e.g API 34 (Android 14) 34

**Comment by XavierJDN** [comment]
did you fix the issue?

**Comment by GalWender** [comment]
I have the same issue

**Comment by matinzd** [comment]
I can see that you are on the old version of react native. Please upgrade to the latest patch version of 0.72. It shouldn't be a huge upgrade. If that didn't fix the issue, let me know. 
Also make sure to follow the documentation for the android setup.

**Comment by Anand8293** [comment]
Hi @matinzd I'm already using patch version of 0.72+ but still same, but with latest react-native 0.75+ working fine

but can't upgrade my long running app to 0.75+, 

**so please help me with 0.72+**

**Comment by nhaduongc** [comment]
I got the issue 
I fixed by upgrade from 0.72.4 to 0.72.17 (I think it is not the reason)
and then I follow https://developer.android.com/codelabs/health-connect 
to config for android app

**Comment by matinzd** [comment]
> Hi @matinzd I'm already using patch version of 0.72+ but still same, but with latest react-native 0.75+ working fine
> 
> but can't upgrade my long running app to 0.75+,
> 
> **so please help me with 0.72+**

Please upgrade to the latest patch of 72 and that should solve your issue. As you can see @nhaduongc is on the latest patch. Hope that's resolved by now. I am going to close this.
